style.css

```diff
@@ -33,22 +31,23 @@ body{
  
 /* Handle */
 ::-webkit-scrollbar-thumb {
-  background: #888; 
+  background: #527ECC; 
 }
 
 /* Handle on hover */
 ::-webkit-scrollbar-thumb:hover {
-  background: #555; 
+  background: #1155CC; 
 }
-/*style for custom scrollbar ends*/
 
 /*style for nav menu starts*/
 .navbar{
 	box-sizing: border-box;
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-	background-color: #1155CC;
+	background-image: linear-gradient(#1155CC, #527ECC);
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
#trialled a box shadow to give my nav more depth when scrolling on other elements